Tools/MonoGame.Effect.Compiler/Program.cs

```diff
@@ -13,7 +13,7 @@ public static int Main(string[] args)
         {
             if (!Environment.Is64BitProcess && Environment.OSVersion.Platform != PlatformID.Unix)
             {
-                Console.Error.WriteLine("The MonoGame content tools only work on a 64bit OS.");
+                Console.WriteLine("The MonoGame content tools only work on a 64bit OS.");
```

Comment:
@harry-cpp what was the idea here?

The reason i had these going to stderr was so that errors appear in the pipeline output.  Removing this means if for whatever reason these fire when running mgfxc from the pipeline then we'll have a failure with no error message.

On Windows at least console output shows both stdout and stderr.